testsuite/features/support/database_handler.rb

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2024 SUSE LLC.
+# Licensed under the terms of the MIT license.
+require 'redis'
+
+# Database handler to interact with a database
+class DatabaseHandler
```

Comment:
Nitpicking a bit.

This class seems to be really specific for a key-value store, Redis in particular.
DatabaseHandler makes me think at something that's supposed to interact with e.g PostgreSQL, or act as a common "interface" for specific implementations.
I'd probably just call it RedisHandler.

Reply:
I renamed DatabaseHandler to be more generic, as for MetricsCollectorHandler.
So, in case we change the implementation in the future. 😅